src/main/resources/data/computercraft/lua/rom/programs/edit.lua

```diff
@@ -101,9 +126,11 @@ local function save(_sPath)
     local function innerSave()
         file, fileerr = fs.open(_sPath, "w")
         if file then
+            if bRun then file.write(runHeader) end
```

Comment:
Using a boolean here feels a little bit weird. Maybe instead we should accept a callback, and do something like:

```lua
if file then
  write(file)
end
```

Then when calling it do this for the "Save" command:

```lua
save(sPath, function(x)
  for _, sLine in ipairs(tLines) do
    file.write(sLine .. "\n")
  end
end)
```

And this for the "Run" command:

```lua
save(sTempPath, function(file)
  file.write(runHandler:format(table.concat(tLines, "\n"), "@" .. fs.getName(sPath)))
end)